assets/scss/helpers/_classes.scss

```diff
@@ -0,0 +1,130 @@
+.micro {
+  font-size: 11px;
+  line-height: 13px;
+  color: $dp-color-grey;
+  text-transform: uppercase;
+}
+/* TEXT COLORS - grey */
+.c-dark-grey {
+  color: $dp-color-darkgrey;
+  /* text */
+}
+
+.c-grey {
+  color: $dp-color-grey;
+  /* icons fonts */
+}
+
+.c-light-grey {
+  color: $dp-color-lightgrey;
+  /* placeholders, hovers*/
+}
+/* VERTICAL SPACES */
+.space-0 {
+  margin-bottom: 0;
```

Comment:
space-b-0 i think it's more descriptive(it's the same for the rest)

Reply:
También viene de otrop lado, yo dejaría ese renombrado para después del lanzamiento.

Si es fácil también se puede hacer ahora, pero ojo, cuiden la compatibilidad hacia atrás o actualizen toda la webapp sin romper nada.